ServerCore/Pages/Events/Standings.cshtml

```diff
@@ -33,6 +33,11 @@
                             @Html.DisplayNameFor(model => model.Teams[0].Score)
                         </a>
                     </th>
+                    <th>
+                        <a asp-page="./Standings" asp-route-sort="@(Model.SortForColumnLink(StandingsModel.SortOrder.HintsUsedAscending, StandingsModel.SortOrder.HintsUsedDescending))">
+                            Hints Used
```

Comment:
This would be the number of tickets, not the number of hints. It's an interesting point though... which is more interesting to display?

Reply:
I'll display tickets, will clarify the title.